src/css/variables.css

```diff
@@ -47,6 +47,7 @@
   --branding-font-size:             1.50em; /* the title, e.g., "itwêwina" */
   --branding-subtitle-font-size:    0.75em; /* the subtitle, e.g., "Plains Cree Dictionary" */
   --definition-title-size:          2.00em;
+  --reference-to-lemma-size:             1.5em;
```

Comment:
Nitpick: this single line is spaced differently than the rest. Could you align it with all the others? I won't block merge if you don't lol